.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+**What is the purpose of this pull request? (put an "X" next to item)**
+
+- [ ] Documentation update
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Other, please explain:
+
+**What changes did you make? (Give an overview)**
+
+**Which issue (if any) does this pull request address?**
+
+**Is there anything you'd like reviewers to focus on?**
+
+**Requirements**
+
+- [ ] I have read the [Code of Conduct](/CODE_OF_CONDUCT.md) and [contributing guidelines](/CONTRIBUTING.md)
+- [ ] I have read and agree to the [Developer Certificate of Origin](https://developercertificate.org/)
```

Comment:
I read this over and realized its actually **kinda scary**, maybe we shouldn't add this because it might scare people into not contributing haha

Reply:
Haha I can pull this part off.